pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -455,3 +455,34 @@ func fixCrcNetworkManagerConfig() (bool, error) {
 	logging.Debug("NetworkManager configuration fixed")
 	return true, nil
 }
+func checkNetworkManagerInstalled() (bool, error) {
+	logging.Debug("Checking if 'nmcli' is available")
```

Comment:
The 'IsRunning' check is enough, if it's not installed, it won't be running, so we don't really need a separate "IsInstalled" check.

Reply:
Some time NetworkManager is installed but service is not running, so I think having both should make sense.

Reply:
Yes, we need to check if it's running. But since it's not going to be running if it's not installed, I would only keep the "running" check, and remove the "installed" check.

Reply:
@cfergeau but if it installed but not running, same as libvirt we have as test or you mean since this is manual so having only one test and tell user to do those manually?

Reply:
Yes, in the libvirt case, we can fix things when libvirt is not installed/not running. In the networkmanager case, we don't fix anything, we only need to detect something is amiss. So a single check would in my opinion be plenty enough.